cluster/config-defaults.yaml

```diff
@@ -335,17 +335,10 @@ skipper_ingress_routegroup_crd_require_hosts: "true"
 skipper_open_policy_agent_enabled: "false"
 skipper_open_policy_agent_styra_token: ""
 
-#
 # FabricGateway controller config
-#
-# fabric_gateway_controller_mode:
-#   - disabled: scales controller to zero replicas
-#   - production: runs the controller
-#
-fabric_gateway_controller_mode: "disabled"
```

Comment:
Should we keep this to disable controller just in case. The same could be achieved by  scaling to 0 while cluster updates are blocked.
OTOH we never used it after rollout.

Reply:
Then lets keep `fabric_gateway_controller_mode` as is and set to "production" by default.
We do not want more than one controller running anyway and controller may have multiple worker threads if needed (not used currently).